src/components/DefaultManifestTable/Table.js

```diff
@@ -31,29 +35,53 @@ Headers.propTypes = {
   columnIndexs: PropTypes.array.isRequired
 }
 
-const Rows = ({ rowCount, columnIndexs }) => {
+const Rows = ({ rowCount, columnIndexs, trPropsHandler, tdPropsHandler }) => {
   const rowIndexs = useMemo(() => integerSequence(rowCount), [rowCount])
 
   return (
     <tbody>
-      {rowIndexs.map(i => <Row key={i} rowIndex={i} columnIndexs={columnIndexs} />)}
+      {rowIndexs.map(i => <TableRow key={i} rowIndex={i} columnIndexs={columnIndexs} trPropsHandler={trPropsHandler} tdPropsHandler={tdPropsHandler} />)}
     </tbody>
   )
 }
 
 Rows.propTypes = {
   rowCount: PropTypes.number.isRequired,
-  columnIndexs: PropTypes.array.isRequired
+  columnIndexs: PropTypes.array.isRequired,
+  trPropsHandler: PropTypes.func.isRequired,
+  tdPropsHandler: PropTypes.func.isRequired
 }
 
-const Row = ({ rowIndex, columnIndexs }) =>
-  <tr>
-    {columnIndexs.map(columnIndex => <td key={columnIndex + '.' + rowIndex}><Cell columnIndex={columnIndex} rowIndex={rowIndex} /></td>)}
-  </tr>
+const TableRow = ({ rowIndex, columnIndexs, trPropsHandler, tdPropsHandler }) => {
+  const state = useManifest()
+  const row = state.rows[rowIndex]
+  return (
+    <tr {...trPropsHandler({ rowIndex, row })}>
```

Comment:
I wish there was a way to ensure this function always returns an object.  I see that the default is an EMPTY_PROPS object, but since it is just a function that can be passed in it feels more fragile than I would hope.

I don't know the best solution though.  Maybe just updating the ReadMe to explain use would be good.

Reply:
Could add an error log if the return is the wrong type of that's a big concern.

Reply:
true